packages/nft-checkout/src/scripts/generate-liqudity-pairs-avalanche.js

```diff
@@ -0,0 +1,76 @@
+const fetcher = require('@traderjoe-xyz/sdk')
+const { Fetcher } = require('@distributedlab/fetcher')
+const { Token: TJToken } = require("@traderjoe-xyz/sdk/dist");
+const { ethers } = require("ethers")
+const fs = require('fs');
+const DEFAULT_FETCHER_CONFIG = {
+  credentials: 'omit',
+  referrerPolicy: 'no-referrer',
+}
+const FUJI_CHAIN = 43113
+const AVALANCHE_CHAIN = 43114
+const AVALANCHE_RPC_URL = 'https://avalanche.public-rpc.com'
+const FUJI_RPC_URL = 'https://rpc.ankr.com/avalanche_fuji'
+
+const getProvider = (chain) => chain === FUJI_CHAIN ? new ethers.providers.JsonRpcProvider(FUJI_RPC_URL) :
+  new ethers.providers.JsonRpcProvider(AVALANCHE_RPC_URL)
+const generatePairs = async (tokenListUrls) => {
+  let tokenLists= []
+
+  for (const url of tokenListUrls) {
+    try {
+      const response = async (rawUrl) => {
+        const url = new URL(rawUrl)
+
+        const fetcher = new Fetcher({
+          baseUrl: url.origin,
+          ...DEFAULT_FETCHER_CONFIG,
+        })
+        return fetcher.get(url.pathname)
+      }
+      let tokenList = (await response(url)).data
+      tokenLists = [...tokenLists, ...tokenList.tokens]
+    } catch (error) {
+      throw new Error(error)
+    }
+  }
+    tokenLists = Array.from(new Map(tokenLists.map(item => [item.address, item])).values());
+  let resultArray =[]
+    for(const tokenA of tokenLists){
+      for(const tokenB of tokenLists){
+        if(tokenA.address !== tokenB.address && tokenA.chainId === tokenB.chainId && (tokenA.chainId === AVALANCHE_CHAIN || tokenA.chainId === FUJI_CHAIN)){
```

Comment:
do this before   `tokenA.chainId === tokenB.chainId && (tokenA.chainId === AVALANCHE_CHAIN || tokenA.chainId === FUJI_CHAIN` instead of doing this on each iteration, also u need create pairs only for one raw at time